### ISSUE TITLE
Change number formatting?

### ISSUE THREAD

**Issue by Exr0n** [xamut/tmux-network-bandwidth#4]
Hi, I have another question which is hopefully legitimate this time :/

Is there any way to change the formatting of the numbers? I find it a little distracting when the numbers change length because it shifts everything around in my status line.

![output](https://user-images.githubusercontent.com/25516241/77873282-5c285380-71fe-11ea-9a8c-dbf9dd23759e.gif)

**Comment by ilya-manin** [comment]
Hey @Exr0n, now you can customize the width with `tmux-network-bandwidth-padding` option.
https://github.com/xamut/tmux-network-bandwidth#customization

**Comment by kamilmirza** [comment]
added
`set-option -g @tmux-network-bandwidth-padding 15`
but fixed padding does not work
updated the plugin already

**Comment by ilya-manin** [comment]
@kamilmirza I just checked and it works fine with this Dockerfile:
```
FROM debian:buster

RUN apt-get update -qq && apt-get install -y \
      curl \
      gawk \
      git-core \
      net-tools \
      tmux

RUN echo "set -g @plugin 'tmux-plugins/tpm'\n\
set -g @plugin 'xamut/tmux-network-bandwidth' \n\
run -b '~/.tmux/plugins/tpm/tpm' \n\
set-option -g status-interval 3 \n\
set-option -g status-right '#{network_bandwidth}' \n\
set-option -g @tmux-network-bandwidth-padding 15" > ~/.tmux.conf

RUN git clone https://github.com/tmux-plugins/tpm ~/.tmux/plugins/tpm
RUN TMUX_PLUGIN_MANAGER_PATH="~/.tmux/plugins" ~/.tmux/plugins/tpm/bin/install_plugins

CMD tmux -u # Force UTF8 support
```

![image](https://user-images.githubusercontent.com/451082/78102129-ba7a4f80-7413-11ea-888b-5381a412ac94.png)

Please double check your config file.

**Comment by kamilmirza** [comment]
apologies!
padding option showed results after I increased
`set -g status-right-length` from `100` to `120`
but its not fixed, here the screenshot:
![2020-04-01_17-27-01](https://user-images.githubusercontent.com/6561475/78137508-e232dd00-743e-11ea-8308-49970656843d.gif)

**Comment by ilya-manin** [comment]
@kamilmirza yeah, fixed at #8

**Comment by kamilmirza** [comment]
fine and dandy now!
